README.md

```diff
@@ -4,6 +4,7 @@ Deadline is a to-do list webapp in a group setting to boost productivity!
 
 ## Virtual Environment
 We recommend doing server development in a Python3.7 environment.
+Everything should be run in the Deadlinee root directory.
```

Comment:
Needs a bit more detail.
eg: what configuration files should the project root directory have for it to run properly?
Usually if the project needs a config.txt, people add config.txt.example to the directory with all the required variables (with fake values and keys). This can then be copied and the variables properly changed to run the project.

Reply:
I think I meant to say the following commands need to be run in the root directory for proper creation and activation of the virtual environment.